bundles/k3d-demo/uds-bundle.yaml

```diff
@@ -18,6 +18,14 @@ packages:
   - name: uds-core #for now
     repository: ghcr.io/defenseunicorns/packages/uds/core
     ref: "0.22.0-upstream"
+    overrides:
+      keycloak:
+        keycloak:
+          variables:
+            - name: INSECURE_ADMIN_PASSWORD_GENERATION
```

Comment:
Can we change the README to mention this setting as well for the dev bundle?

Reply:
done